modules/members.js

```diff
@@ -0,0 +1,101 @@
+// @ts-check
+
+const Augur = require("augurbot-ts"),
+  u = require("../utils/utils"),
+  c = require("../utils/modCommon"),
+  Discord = require("discord.js"),
+  config = require("../config/config.json");
+
+/**
+ * Creates a profile card - a PNG that contains some user information in a fun format!
+ * @param {Discord.GuildMember} member The member to create the profile card for.
+ * @returns {Promise<Buffer|undefined>} File-like object to attach to your response.
+ */
+async function makeProfileCard(member) {
+  try {
+    const badgeData = require("../utils/badges"),
+      RankInfo = require("../utils/RankInfo"),
+      Jimp = require("jimp");
+
+    const badgePath = "./media/badges/";
+    const font = await Jimp.loadFont(Jimp.FONT_SANS_16_WHITE);
+    const card = await Jimp.read("./media/background.jpg");
+
+    const members = member.guild.members.cache;
+    const rank = await u.db.user.getRank(member.id, members);
+    const badges = await badgeData(member.roles.cache);
+
+    const avatar = await Jimp.read(member.displayAvatarURL({ size: 64, extension: "png" }));
+
+    card.blit(avatar, 8, 8)
+      // eslint-disable-next-line no-control-regex
+      .print(font, 80, 8, member.displayName.replace(/[^\x00-\x7F]/g, ""), 212)
+      .print(font, 80, 28, "Joined: " + (member.joinedAt ? member.joinedAt.toLocaleDateString("en-US", { year: 'numeric', month: 'long', day: 'numeric' }) : "???"), 212);
```

Comment:
This seems a bit clearer
```suggestion
      .print(font, 80, 28, "Joined: " + (member.joinedAt ? u.moment(member.joinedAt).format("MMMM D, YYYY") : "???"), 212);
```

Reply:
Done